### ISSUE TITLE
Consider supporting Sdkman

### ISSUE THREAD

**Issue by odisseus** [propensive/fury-old#1281]
[Sdkman](https://sdkman.io/) is a tool that manages various development environments. I just learned about its existence, but it seems to have [attracted](https://github.com/sdkman/sdkman-cli/stargazers) some interest.

It would be nice if that tool allowed the users to install Fury along with Java, IPFS and any other dependencies.

**Comment by propensive** [comment]
Agreed. Having Fury install its own versions of these dependencies should be a last resort. We can add Graal Native Image to that list. The one exception is Java, where we may need to ensure an exact minor version because a build depends on it (#158) but that's not even a feature of Fury yet.

**Comment by veyga** [comment]
I use this tool alot for messing around with different scala and Java versions, it's neat. Sdkman could be done in the near future, but I think that a docker image is a must. I think the average user is more comfortable with tinkering a new piece of software if its containerized, rather than adding more dotfiles to home directories.
